conf/modules/markduplicates.config

```diff
@@ -91,42 +91,42 @@ process {
     }
 
     withName: 'GATK4SPARK_MARKDUPLICATES' {
-        ext.args         = '--remove-sequencing-duplicates false -VS LENIENT'
-        ext.prefix       = { "${meta.id}.md.cram" }
-        publishDir       = [
-            enabled: !params.save_output_as_bam,
+        ext.args   = '--remove-sequencing-duplicates false -VS LENIENT'
```

Comment:
Missing curly brackets

Reply:
```suggestion
        ext.args   = { '--remove-sequencing-duplicates false -VS LENIENT' }
```

Reply:
we dont' really need them in that case, but let's be consistent